src/index/IndexImpl.Text.cpp

```diff
@@ -916,7 +916,7 @@ Index::WordEntityPostings IndexImpl::getContextEntityScoreListsForWords(
           getWordPostingsForTerm(terms[onlyWordsFrom]);
       Index::WordEntityPostings eWep =
           getEntityPostingsForTerm(terms[useElFromTerm]);
-      resultWep = FTSAlgorithms::intersect(wWep, eWep);
+      resultWep = FTSAlgorithms::crossIntersect(wWep, eWep);
```

Comment:
Is it very difficult to also make this work for more than two terms by also updating the `intersectKWay`?